ctagsdotd/vbnet.ctags

```diff
@@ -0,0 +1,12 @@
+# See additional-language.ctags for maintaining this file
+--langdef=vbnet
+--langmap=vbnet:.vb
+--regex-vbnet=/^(?i)\s*(?:(?:Public|Private|Protected|Friend|Overloads|Overrides|Overridable|NotOverridable|MustOverride|Shared|Shadows|Async|Iterator)\s+)*\s*(?:Function|Sub)\s+([A-Z_0-9\[\]]*)\(.*?\)\s*(?:As\s+[\w.]+)?/\1/m,method/
```

Comment:
(I have ~0 Visual Basic knowledge). Could you explain why this is using `m,method` instead of `f,function`? For example, Rust supports both methods and functions, so the output of `ctags --list-kinds-full=Rust` has both `P method` and `f function`.

Reply:
There are actually two types in VB.NET Subs and Functions. The difference between them is that Functions have a return value while Subs do not. And since the [language-specification] (https://learn.microsoft.com/en-us/dotnet/visual-basic/reference/language-specification/type-members)calls both methods, I opted for that one.